Client/core/CSettings.cpp

```diff
@@ -3325,10 +3327,12 @@ void CSettings::SaveData()
     // High detail vehicles (just set cvar, real change occur on next connect)
     bool bHighDetailVehicles = m_pCheckBoxHighDetailVehicles->GetSelected();
     CVARS_SET("high_detail_vehicles", bHighDetailVehicles);
+    gameSettings->ResetVehiclesLODDistance();
 
     // High detail peds (just set cvar, real change occur on next connect)
     bool bHighDetailPeds = m_pCheckBoxHighDetailPeds->GetSelected();
     CVARS_SET("high_detail_peds", bHighDetailPeds);
+    gameSettings->ResetPedsLODDistance();

```

Comment:
Does this only apply the change if not connected?

Reply:
Right now it will apply the change when pressing the "Save" or "Default" button. Also client does not have to be connected.

Reply:
It shouldn't apply if script has changed the setting